src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,21 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
+  name: "exfmt"
+  link: "https://github.com/lpil/exfmt"
+  isPreInstalled: false
+
+  options: {
+    Elixir: true
```

Comment:
Should be `false` since no options are supported. Although, no options exist, however if in the future options were added to the language, at least this would not incorrectly report in the documentation that `exfmt` supported those new options 😉 .